single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultSVGWriter.java

```diff
@@ -870,9 +870,10 @@ private void drawFeederInfo(String prefixId, FeederNode feederNode, List<Point>
 
         String svgId = escapeId(feederNode.getId()) + "_" + feederInfo.getComponentType();
         g.setAttribute("id", svgId);
+        String measurementType = feederInfo.getComponentType();
 
         String side = feederNode.getFeeder() instanceof FeederWithSides ? ((FeederWithSides) feederNode.getFeeder()).getSide().name() : null;
-        metadata.addFeederInfoMetadata(new FeederInfoMetadata(svgId, feederNode.getEquipmentId(), side, feederInfo.getUserDefinedId()));
+        metadata.addFeederInfoMetadata(new FeederInfoMetadata(svgId, feederNode.getEquipmentId(), side, feederInfo.getUserDefinedId(), measurementType));
```

Comment:
One minor thing also, measurementType is quite important in fact, at least mor than userDefinedId for instance, so you can put it more to the front in the arguments list.